doc/JSON_INFO.md

```diff
@@ -473,12 +473,13 @@ The syntax listed here is still valid.
 | `name`           | In-game name displayed.
 | `bash_resist`    | How well a material resists bashing damage.
 | `cut_resist`     | How well a material resists cutting damage.
+| `acid_resist`    | Ability of a material to resist acid. Metal should be 0. Complete resistance is 3.
+| `elec_resist`    | Ability of a material to resist electricity.
+| `fire_resist`    | Ability of a material to resist fire.
+| `chip_resist`    | 
```

Comment:
Description?

Reply:
see above

> Instead of "Ability of a material to resist chipping (like from a chisel?)." what should I write? I'm just going to leave it blank for the moment. Someone who knows what they are talking about can fill it in.

Reply:
`Returns resistance to being damaged by attack against the item itself`

I'd appreciate it if you found this stuff out before making a pr updating the document...

Reply:
Originally chip resist wasn't there at all. It's a good point you raise: there are more than a few identifiers that the doc doesn't include. 

Provided that this isn't intentional for the sake of clarity and brevity, is it better to have a identifier included with no description, or excluded entirely?

Clearly it is best to have both the identifier _and_ the description. But when I notice an identifier that I can't immediately provide an accurate description for, is it better to just leave it absent?

Thanks as always.